pywinauto/hooks.py

```diff
@@ -0,0 +1,312 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_uint, c_long, c_longlong, c_ulong, c_void_p, byref, sizeof
+from ctypes import Structure
+import atexit
+
+if sizeof(POINTER(c_int)) * 8 == 64:
+    hinstance = c_longlong
+else:
+    hinstance = c_long
+
+cmp_func = CFUNCTYPE(c_int, c_int, hinstance, POINTER(c_void_p))
+DWORD = c_ulong
+
+windll.kernel32.GetModuleHandleA.restype = wintypes.HMODULE
```

Comment:
`wintypes.DWORD` and others is a great finding! It means we can eliminate a lot of win32 definitions from `win32defines.py` by using `wintypes`. I'm surprised how we missed it.